src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  place-content: center;
+  grid-template-columns: 200px;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
```

Comment:
Similar to the previous comment, consider defining a `max-width` for the media query starting at 768px to prevent overlap with other media queries and ensure that styles are applied only within the intended range.